### DISCUSSION TITLE
Need help setting up CI to upload my plugin to luarocks

### DISCUSSION THREAD

**Discussion by Klafyvel** [lumen-oss/rocks.nvim#301] [Q&A]
Hi there! I'm switching to rocks.nvim for my plugins. However, I need help debugging the CI. I followed the tutorial [here](https://github.com/nvim-neorocks/sample-luarocks-plugin), but I get an error when I try to update my plugin on luarocks (see [here](https://github.com/Klafyvel/nvim-smuggler/actions/runs/8764307343/job/24330231945)). 

Can you help me please? :)

**Comment by mrcjkb** [comment]
Hey 👋 

It's always awesome to see more plugins being pushed to luarocks! 😄 

It looks like your workflow ran against the main branch. 
If you don't specify a version, or its not triggered by a tag, it will set the version to the branch name (`main` in your case).
The error says that it failed to upload because a rock with that version has already been published.
